pkg/controller/deployment/common.go

```diff
@@ -10,4 +10,11 @@ const (
 	conditionsPrefix  = "CertManager"
 )
 
-var targetVersion = os.Getenv("OPERAND_CERT_MANAGER_IMAGE_VERSION")
+var targetVersion = "dev"
+
+func init() {
+	version := os.Getenv("OPERATOR_VERSION")
+	if len(version) != 0 {
+		targetVersion = version
+	}
```

Comment:
let's migrate this to the startup methods and dependency inject. having mutable package level state can cause issues, see https://peter.bourgon.org/go-for-industrial-programming/